libraries/golang/layering/models/silos.go

```diff
@@ -4,3 +4,27 @@ type Silo struct {
 	Id string `db:"id"`
 	State string `db:"state"` // TODO: Decide possible state values and replace with an enum. (active, disabled, deleted?)
 }
+
+type Node struct {
+	Id string `db:"id"`
+	Doc string `db:"doc"`
+}
+
+type Edge struct {
+	Id string `db:"id"`
+	Txn_id string `db:"txn_id"`
```

Comment:
Steven I noticed ID has type `string` in other places but in the `FxnArgs` struct `FxnId` is `int`. Do you know which I should use for `TxnId`? Is that field going to be be the same as the `FxnId`?